infra/terraform/modules/data_access/github_webhooks_handler/tests/conftest.py

```diff
@@ -0,0 +1,140 @@
+from hashlib import sha1
+from moto import mock_sns
+import boto3
+import hmac
+import json
+import os
+
+import pytest
+
+
+#################
+#  Tests state  #
+#################
+
+
+@pytest.fixture
+def state():
+    return {
+        "event": {
+            "headers": {
+                "X-Hub-Signature": "signature",
+                "X-GitHub-Event": "team",
+            },
+            "body": "{\"a JSON string\": true}",
+        }
+    }
+
+
+###########
+#  Event  #
+###########
+
+
+@pytest.fixture
+def event(state):
+    return state["event"]
```

Comment:
It is actually used in the test to call the `send_to_sns(event, context)` function.